platform_it/src/test/java/com/splicemachine/test/SpliceTestPlatformConfig.java

```diff
@@ -125,6 +125,14 @@ public static Configuration create(String hbaseRootDirUri,
             config.set("hbase.master.keytab.file", keytab);
             config.set("yarn.nodemanager.principal", "yarn/example.com@EXAMPLE.COM");
             config.set("yarn.resourcemanager.principal", "yarn/example.com@EXAMPLE.COM");
+            config.set("splicemachine.enterprise.key","EE_KEY");
```

Comment:
Can you make it so that the enterprise key is not required? Either by picking it automatically from the -ee repo, not enabling LDAP by default, or somehow avoiding the EE check on standalone.

Reply:
I think the easiest method is that we use sed tool to substitute "EE_KEY" string with our EE-key provided from cli.I will try to achieve that.